apps/web/src/components/Notifications/Notification.tsx

```diff
@@ -108,16 +107,16 @@ export function Notification({ notificationRef, queryRef }: NotificationProps) {
       }
     | undefined;
 
+  const { showDrawer } = useDrawerActions();
+
   const handleNotificationClick = useMemo((): NotificationClick => {
     function showUserListModal() {
-      showModal({
+      showDrawer({
         content: (
           <NotificationUserListModal notificationId={notification.id} fullscreen={isMobile} />
         ),
```

Comment:
It feels a little bit strange to have both a `name` and a `content` field. In my head it would be on or the other, like maybe you can infer `content` from the `drawerName` but I'm not sure. 

Are there cases where you show `notificationUserList` with different content?

Reply:
This is so much better. One last thing I'll say is using the `type` for the header text seems like it might break down quickly, but I'm fine with it for now.